### ISSUE TITLE
Need a manual deploy button

### ISSUE THREAD

**Issue by argyleink** [denoland/deploy_feedback#331]
Sometimes the automatic deploys are hours behind a commit. It'd be nice to have a button for a manual git pull and deploy, for times when the automated system is behind.

**Comment by lino-levan** [comment]
100%, reiterating what I said on the discord

> this would be huge, this would fix 50% of my issues with deploy single-handedly

**Comment by ukabu** [comment]
Agreed. And maybe a timestamp of the last check for changes. More than 12 hours since a push and no deploy!?

**Comment by argyleink** [comment]
Wishing for this button again right now
![Screenshot 2024-03-17 at 9 48 33 PM](https://github.com/denoland/deploy_feedback/assets/1134620/24f37f8c-df95-4457-a1b3-dbd949a902fe)

Not sure what internal error happened, but I'd love an easy retry button somewhere.

**Comment by dgreensp** [comment]
Stuff like this sitting and not getting implemented makes me concerned about Deno Deploy as part of our production infrastructure in the long term.
